### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *roaring-nougat-6a9cf4* failed.

### ISSUE THREAD

**Issue by bosszeka40-pixel** [bosszeka40-pixel/Cms-my#7]
### <span aria-hidden="true">❌</span> Deploy Preview for *roaring-nougat-6a9cf4* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 5b0b93c5f759c8b47e9576f8679cb65af4ee9aa6 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/roaring-nougat-6a9cf4/deploys/6a85640a149e8a0007b076dc |

_Originally posted by @netlify in https://github.com/bosszeka40-pixel/Cms-my/pull/6#issuecomment-5339290813_
